### PR TITLE
feature to save as PNG

### PR DESCRIPTION
**Pull request by geekswaroop**
@zudi, I've implemented these functions first, next, I will use these on the post-processing scripts.

### CODE REVIEW

File: connectomics/data/utils/data_io.py [hunk 1]
```diff
@@ -49,7 +54,7 @@ def readimgs(filename):
 
     return data
 
-def writeh5(filename, dtarray, dataset='main'):
+def writeh5(filename, filepath, dtarray, dataset='main', saveAsPng = False):
```
Comment (geekswaroop) [inline connectomics/data/utils/data_io.py:57]: Here, we need a filepath to save the PNG images. Either you could pass the filepath separately or use the filename to strip the H5 filename. I have done the former. Open to any other design changes!

File: connectomics/data/utils/data_io.py [hunk 2]
```diff
@@ -23,6 +23,11 @@ def readvol(filename, dataset=''):
         raise ValueError('unrecognizable file format for %s'%(filename))
     return data
 
+def savevol(filename, vol, dataset='main'):
```
Comment (geekswaroop) [inline connectomics/data/utils/data_io.py:26]: This is the savevol function, which is very similar to writeh5. Is this implementation fine?

File: connectomics/utils/processing/process_syn.py [hunk 3]
```diff
@@ -82,4 +82,14 @@ def polarity2instance(volume, thres=0.5, thres_small=128,
                        int(segm.shape[1]*scale_factors[1]), 
                        int(segm.shape[2]*scale_factors[2]))
         segm = resize(segm, target_size, order=0, anti_aliasing=False, preserve_range=True)
-    return segm
\ No newline at end of file
+    return segm
+
+def main():
+    vol_path = sys.argv[1]
+    format = sys.argv[2]
```
Comment (geekswaroop) [inline connectomics/utils/processing/process_syn.py:89]: Hope these 2 arguments are fine. 
1) vol_path refers to the predicted volume
2) format is for the saving function. H5/PNG.

Usage: `python process_syn.py path/to/file png`

File: connectomics/data/utils/data_io.py [hunk 4]
```diff
@@ -23,10 +23,16 @@ def readvol(filename, dataset=''):
         raise ValueError('unrecognizable file format for %s'%(filename))
     return data
 
-def savevol(filename, vol, dataset='main'):
-    fid = h5py.File(filename, 'w')
-    fid.create_dataset(dataset, data=vol, compression='gzip')
-    fid.close()
+def savevol(filename, vol, dataset='main', format='h5'):
```
Comment (geekswaroop) [inline connectomics/data/utils/data_io.py:26]: I have a doubt here. The filename specifies where you need to save the vol. For ex: `/n/home/test.h5`
If I'm using the PNG format, I don't use this information. Plus, the filename needs to be passed as a parameter in the `process_syn.py` I guess. Let me know if there's anything I can do about this.

**Comment by geekswaroop** [comment]
@zudi-lin I've done as you mentioned. Please check if this is fine. Thanks!

**Comment by zudi-lin** [comment]
Thanks! I have merged this PR

**Pull request merged**